phovea_registry.js

```diff
@@ -4,13 +4,13 @@
  * Licensed under the new BSD license, available at http://caleydo.org/license
  **************************************************************************** */
 
-import {register} from 'phovea_core/src/plugin';
-import reg from './src/phovea';
+import {PluginRegistry} from 'phovea_core';
+import reg from './dist/phovea';
```

Comment:
Is this correct? Shouldn't it be the following for the **source** file:

```suggestion
import reg from './src/phovea';
```